### DISCUSSION TITLE
Suggestion Regarding the Update Speed of Packages in WinGet

### DISCUSSION THREAD

**Discussion by kevincharp** [microsoft/winget-pkgs#185720] [General]
I want to express my concern about the slow update of packages in winget, specifically regarding the Git for Windows package. Recently, while trying to clone several repositories from GitHub and GitLab using SSH, I encountered stalling issues due to using an outdated version of Git. The latest version (v2.47.0.windows.2) was not available through winget, forcing me to manually download the installer.

I understand that winget packages are managed by a third party, which results in delays for updates. However, this situation can cause significant inconveniences for many users, especially those relying on an agile development workflow. The lack of access to the latest versions can lead to conflicts and a loss of productivity.

Due to this issue, many users feel compelled to turn to other package managers, such as Chocolatey or Scoop, which tend to update more quickly. This shouldn’t be the norm, as winget has the potential to be a central tool in the development workflow.

Thank you for considering this suggestion, and I appreciate the effort you all put into maintaining and improving the development tools ecosystem.

Who should I direct this complaint or suggestion to more effectively?

I’m leaving the link to the discussion in the Git for Windows repository where this is mentioned: https://github.com/git-for-windows/git/discussions/5224#discussioncomment-11053833

![image](https://github.com/user-attachments/assets/d2828fbb-1ef2-427e-9168-3126f012e7c4)

**Comment by SpecterShell** [comment]
It's weird that nobody is actively maintaining this package. 🤔

Anyway I'll pick up this package. It'll be automated and should be available within 0.1 - 2 days.
